.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -14,6 +14,8 @@ body:
         required: true
       - label: I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.
         required: true
+      - label: I am an app developer, not an end user. (End users, please report issues to the app developer.)
```

@nikwen I'm bikeshedding a tiny bit here, so not blocking merge on this, but could we maybe say something like:
```suggestion
      - label: This is a bug within Electron itself, and not in a specific application. (Please report issues with specific Electron applications to the app developer.)
```
I just don't want to completely dissuade end users, but I also get what you're saying, we don't want a Slack bug reported here vs. with Slack's support.